pkg/buses/gcppubsub/bus.go

```diff
@@ -40,15 +41,15 @@ type CloudPubSubBus struct {
 	logger *zap.SugaredLogger
 }
 
-func NewCloudPubSubBusDispatcher(ref buses.BusReference, projectID string, opts *buses.BusOpts) (*CloudPubSubBus, error) {
+func NewCloudPubSubBusDispatcher(ref buses.BusReference, busUID string, projectID string, opts *buses.BusOpts) (*CloudPubSubBus, error) {
 	ctx := context.Background()
 	pubsubClient, err := pubsub.NewClient(ctx, projectID)
 	if err != nil {
 		return nil, err
 	}
 
 	bus := &CloudPubSubBus{
-		ref:          ref,
```

Comment:
what is ref in this context? It looks like it is being passed in but I would assume a ref to a bus would point to the object that is being created with `&CloudPubSubBus{...}`